### PR TITLE
[ARM porting] Required code changes to fix build issues

### PR DESCRIPTION
**Pull request by vimleshsgt**
* Define PAGE_SIZE, PAGE_SHIFT
* Change arch from x86_64 to aarch64
* temporarily added MODULE_LICENSE()

### CODE REVIEW

**Comment by huanghua78** [comment]
Do we have a test bed for ARM port testing?
What is the test result, @vimleshsgt ?

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by nezzzu** [comment]
@huanghua78 This is needed for the Sage project Arm porting. The patches that @vimleshsgt and @Nalini-Seagate are submitting are tested on the Sage cluster Arm machines and approved by me.

Notice that these individual patches need to be merged first to their individual branches then to common sage-arm-port branch.

**Comment by huanghua78** [comment]
> @huanghua78 This is needed for the Sage project Arm porting. The patches that @vimleshsgt and @Nalini-Seagate are submitting are tested on the Sage cluster Arm machines and approved by me.
> 
> Notice that these individual patches need to be merged first to their individual branches then to common sage-arm-port branch.

Thank you, @nezzzu . Please review and approve this PR.

**Review by nezzzu — APPROVED** [review]

File: motr/linux_kernel/m0tr_main.c [hunk 1]
```diff
@@ -64,7 +64,9 @@ module_exit(motr_exit);
  * marker is necessary to remove the warnings, keeping this blank to make
  * compiler happy.
  */
-MODULE_LICENSE();
+
+/* Added GPL per suggestions to avoid compilation error, may need to be reviewed */
+MODULE_LICENSE("GPL");
```
Comment (huanghua78) [inline motr/linux_kernel/m0tr_main.c:69]: This should be a temporary solution, and be fixed later.

**Review by huanghua78 — APPROVED** [review]
Looks fine.

**Comment by huanghua78** [comment]
* Define PAGE_SIZE, PAGE_SHIFT
* Change arch from x86_64 to aarch64
* Temporarily added MODULE_LICENSE()

**Review by madhavemuri — COMMENTED** [review]
I would suggest make this patch workable in main branch too for both x86 and arm architectures.

**Comment by huanghua78** [comment]
> I would suggest make this patch workable in main branch too for both x86 and arm architectures.

Of course that is needed.
I suppose that would be done in another task.

File: motr/linux_kernel/m0tr_main.c (hunk 1 above)
Comment (andriytk) [reply motr/linux_kernel/m0tr_main.c:69]: Solution to what problem? Just curious.

File: lib/user_space/user_x86_64_atomic.h [hunk 2]
```diff
@@ -182,10 +182,10 @@ static inline bool m0_atomic64_dec_and_test(struct m0_atomic64 *a)
 }
 
 
-static inline bool m0_atomic64_cas(volatile int64_t * ptr, int64_t old, int64_t newval)
+static inline bool m0_atomic64_cas(int64_t * ptr, int64_t old, int64_t newval)
```
Comment (andriytk) [inline lib/user_space/user_x86_64_atomic.h:185]: Why is this change needed?
